INSTALLATION-macOS-Sequoia-Intel.md

```diff
@@ -0,0 +1,104 @@
+# Procédure d’installation de `mdl` (markdownlint) sur macOS Sequoia (Intel)
+
+Ce guide explique comment installer la version Ruby de `mdl` (markdownlint) sur un Mac équipé d’un processeur Intel, en utilisant un environnement propre et moderne avec `rbenv`.
+
+## Prérequis
+
+- Disposer d’un terminal
+- Avoir les droits d’administration sur la machine
+
+## Étapes détaillées
+
```

Comment:
Please keep our primary documentation in English (translations to other languages are welcome).

Moreover, if you're going to add installation instructions, move this to docs/ and link to it from the `Installation` section of `README.md`